sky/packages/sky/lib/widgets/mimic.dart

```diff
@@ -116,19 +154,12 @@ class Mimicable extends StatefulComponent {
     });
   }
 
-  void _startMimicIfNeeded() {
-    if (_didStartMimic)
-      return;
-    assert(_mimic != null);
-    Point globalPosition = localToGlobal(Point.origin);
-    _mimic._startMimic(key, globalPosition & _size);
-    _didStartMimic = true;
-  }
-
   Widget build() {
-    if (_mimic != null) {
-      _startMimicIfNeeded();
-      return new ConstrainedBox(constraints: new BoxConstraints.tight(_size));
+    _readyForMimic = _mimicRequested && _size != null;
```

Comment:
"ready" is a weird term to use here. One would expect a Mimicable to be Ready before it's Requested.